### PR TITLE
Clean up bazel base test case proxy methods

### PR DESCRIPTION
**Pull request by maximn**

### CODE REVIEW

**Comment by maximn** [comment]
@ittaiz

File: java/build/bazel/tests/integration/BazelBaseTestCase.java [hunk 1]
```diff
@@ -17,13 +17,8 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 
-import java.io.File;
 import java.io.IOException;
-import java.util.List;
 
-/**
```
Comment (ittaiz) [inline java/build/bazel/tests/integration/BazelBaseTestCase.java]: Why did you remove this? I think the comment still holds

Comment (maximn) [reply java/build/bazel/tests/integration/BazelBaseTestCase.java]: `A base class to do integration test that call Bazel` - I don't feel like it adds useful information and it doesn't really call Bazel anymore.

**Comment by maximn** [comment]
regarding  “contents” -> “workspaceDirectoryContents”
Let's talk about the name f2f. Also if we change it let's consider changing the signature to return `List<Path>` instead of `List<String>`

File: java/build/bazel/tests/integration/BazelBaseTestCase.java (hunk 1 above)
Comment (ittaiz) [reply java/build/bazel/tests/integration/BazelBaseTestCase.java]: I think you’re misreading it. It’s a base class to do an integration test which calls bazel. The base doesn’t call it. The test calls it.

Comment (maximn) [reply java/build/bazel/tests/integration/BazelBaseTestCase.java]: Do you see value in this comment? Is it just project's style?

**Comment by ittaiz** [comment]
Just project’s style.
